app/actions/actionTypes.tsx

```diff
@@ -244,6 +244,10 @@ export function createAction<T extends { type: ACTION_TYPES }>(d: T): T {
   return d;
 }
 
+interface GetMultiPapersInCollection extends CommonPaginationResponsePart {
```

Comment:
CommonPaginationResponsePart is deprecated.
Please use `RawPaginationResponseV2` or `PaginationResponseV2`